react/training/src/scss/_variables.scss

```diff
@@ -0,0 +1,8 @@
+// Colors
```

Comment:
Let split all this const into a `_variables/colors.scss` and ` and `_variables/fontStyles.scss` files